src/Parser/Core/Modules/ConcordanceUptimeTracker.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+const debug = false;
+const CONCORDANCE_SPELLS = {
+  INTELLECT: SPELLS.CONCORDANCE_OF_THE_LEGIONFALL_INTELLECT,
+  AGILITY: SPELLS.CONCORDANCE_OF_THE_LEGIONFALL_AGILITY,
+  STRENGTH: SPELLS.CONCORDANCE_OF_THE_LEGIONFALL_STRENGTH,
+  VERSATILITY: SPELLS.CONCORDANCE_OF_THE_LEGIONFALL_VERSATILITY,
+};
+class ConcordanceUptimeTracker extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  appliedBuff = null;
```

Comment:
This is only set if statistic is called, since this isn't guaranteed and it's not an event handler it is kinda unreliable. Would it be an idea to either remove this property or making it a getter? I'd prefer making it a getter; this separates the code leaving only display responsibilities to the statistic method.

Reply:
So moving this code up into a getter would be fine for you? No problemo.